README.md

```diff
@@ -163,8 +166,8 @@ The automation requires an HTTP or NFS server to hold the ICP binaries and docke
 | name | required                        | value        |
 |----------------|------------|--------------|
 | `vsphere_server`   | yes          | IP or hostname of vSphere server |
-| `vsphere_user`   | yes          | Username for vSphere server |
-| `vsphere_password`     | yes          | Password for vSphere user     |
+| `vsphere_user`   | no, moved to environment variables          | Username for vSphere server |
+| `vsphere_password`     | no, moved to environment variables          | Password for vSphere user     |
```

Comment:
we also need to add information here in the README for the new `ssh_password` (and delete the `ssh_key`) and reflect these on the `terraform.tfvars` example in the README